Gemfile

```diff
@@ -71,5 +71,6 @@ group :test do
   gem 'selenium-webdriver'
```

Comment:
![image](https://github.com/BranBayou/ror-blog-app/assets/93039359/46600692-ce8e-44a3-a675-a94d7632f4af)

- [ ] It would be nice if you fix the highlighted link in the readme file shown in the image above 👀 because it is broken 😨 Kindly fix it or remove it **(If you don't have any dedicated section for the broken link)** to make your readme descriptive and professional 👍

Reply:
Hello @HAMMAS-SALEEM, I don't understand what is broken here. Can you elaborate on this?

Reply:
Hey @HAMMAS-SALEEM, I fixed it.